caffe2/core/tensor.cc

```diff
@@ -105,25 +105,6 @@ GetInt8TensorInfo(const void* c, size_t* capacity, DeviceOption* device) {
   return GetTensorInfo(&(int8_tensor->t), capacity, device);
 }
 
-// since we only have one tensor, probably need to remove this at some point?
-static CaffeMap<TypeIdentifier, TensorInfoCall> tensor_info_call_registry_{
-    {TypeMeta::Id<Tensor>(), GetTensorInfo},
-    {TypeMeta::Id<int8::Int8TensorCPU>(), GetInt8TensorInfo},
```

Comment:
looks like we still have `int8::Int8TensorCPU` before we can get rid of this. I believe caffe2 int8 tensor is still in use in production. cc @dskhudia do you know if this is true?